hnn_core/tests/test_compare_hnn.py

```diff
@@ -78,3 +79,13 @@ def test_mpi():
 def test_joblib():
     """Test that running hnn-core with Joblib context manager when n_jobs=2."""
     run_hnn_core(backend='joblib', n_jobs=2)
+
+
+def test_num_trials():
+    """Test that running 0 trials retuns an exception."""
+    hnn_core_root = op.dirname(hnn_core.__file__)
+    params_fname = op.join(hnn_core_root, 'param', 'default.json')
+    params = read_params(params_fname)
+    net = Network(params)
+    with pytest.raises(ValueError, match="Invalid number of simulations: 0"):
+        simulate_dipole(net, n_trials=0)
```

Comment:
thanks for adding the test. Since `simulate_dipole` is part of `dipole.py` I think it makes more sense to add this test to `test_dipole.py`.

Also you have a merge conflict, probably the branch is too old.